### DISCUSSION TITLE
导航栏前面的图标怎么加

### DISCUSSION THREAD

**Discussion by Duanjyy** [Tokinx/Wing#13] [问答]
就是导航栏前面的图标怎么加
![image](https://user-images.githubusercontent.com/91202296/170947032-ceee99b8-2d34-4429-bf84-5a022d8614b4.png)

**Comment by Tokinx** [comment]
更新了wiki，你可以参考进行设置：https://github.com/Tokinx/Wing/wiki/%E5%A6%82%E4%BD%95%E6%B7%BB%E5%8A%A0%E5%AF%BC%E8%88%AA%E5%9B%BE%E6%A0%87
